### DISCUSSION TITLE
Resource Clickable

### DISCUSSION THREAD

**Discussion by amcanlubo** [vkurko/calendar#62] [Ideas]
Hi! great docs!

I'd like to ask if it's possible to make each resources clickable on this calendar to run some functions?
Also how to get resource properties?

Thanks  in advance!

**Comment by vkurko** [comment]
Hi @amcanlubo

Thanks for your question. It appears that the Event Calendar currently lacks suitable settings for this. I think that FullCalendar's [settings](https://fullcalendar.io/docs/resource-area-header-render-hooks) for this purpose would just be helpful. I will try to implement them in the near future.

**Reply by amcanlubo** [reply]
Thank you! looking forward to it!

**Comment by vkurko** [comment]
@amcanlubo 
In v0.14.0 there are 2 new options [resourceLabelContent](https://github.com/vkurko/calendar#resourcelabelcontent) and [resourceLabelDidMount](https://github.com/vkurko/calendar#resourcelabeldidmount). Please check.

**Reply by amcanlubo** [reply]
Alright will try this out! Thank you! :D
